PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.cxx

```diff
@@ -3993,26 +4017,3 @@ void AliAnalysisTaskGammaCaloMergedML::GetClusterReadout(AliVCluster* cluster, A
   //std::cout<<"\n";
   //}
 }
-
-void AliAnalysisTaskGammaCaloMergedML::ResetBuffer(){
```

Comment:
I can only see a fraction of the code, but since you removed the ResetBuffer function: do you ensure that the tree buffer is reset properly elsewhere now?